frontend/.env.development

```diff
@@ -14,8 +14,12 @@ SENDY_API_KEY=
 SENDY_API_URL=
 SENDY_LIST_ID=
 
-NEXT_PUBLIC_API_URL=http://localhost:8080
```

Comment:
We can drop the `NEXT_PUBLIC_` prefix since we're only calling things from the Next server side now